modules/connectors/ninja_van/karrio/mappers/ninja_van/settings.py

```diff
@@ -0,0 +1,24 @@
+"""Karrio Ninja Van client settings."""
+
+import attr
+import karrio.providers.ninja_van.utils as provider_utils
+
+
+@attr.s(auto_attribs=True)
+class Settings(provider_utils.Settings):
+    """Ninja Van connection settings."""
+
+    # required carrier specific properties
+    api_key: str = None
+    secret_key: str = None
+    account_number: str = None
+    track_api_key: str = None
+    track_secret_key: str = None
```

Comment:
According to the NinjaVan API, here is what you need for the [Oauth login](https://api-docs.ninjavan.co/en#tag/OAuth-API/paths/~1%7BcountryCode%7D~12.0~1oauth~1access_token/post)

```
{
  "client_id": "string",
  "client_secret": "string",
  "grant_type": "client_credentials"
}
```

So 

```suggestion
    client_id: str = None
    client_secret: str = None
```